incubator/nodejs-loopback/image/Dockerfile-stack

```diff
@@ -0,0 +1,34 @@
+FROM appsody/nodejs:0.2
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+ENV APPSODY_DEPS=/project/user-app/node_modules
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/node_modules
+ENV APPSODY_WATCH_REGEX="^.*.js$"
```

Comment:
should we be watching `.ts` files here? my understanding is that we need to watch .ts files, otherwise, it will not automatically rebuild your TS? and the .js files would stay the same but I could be wrong?